lib/stoplight/light/runnable.rb

```diff
@@ -21,12 +19,12 @@ def state
       #
       # @return [String] returns current light color
       def color
-        failures, state = failures_and_state
+        failures, state = config.data_store.get_all(config)
```

Comment:
I think we should hide `config.data_store.get_all(config)` behind some private method. WDYT?

Reply:
I'm not sure, why?